src/servers/src/error.rs

```diff
@@ -463,11 +463,11 @@ impl From<Error> for tonic::Status {
         // (which is a very rare case), just ignore. Client will use Tonic status code and message.
         let status_code = err.status_code();
         if let Ok(code) = HeaderValue::from_bytes(status_code.to_string().as_bytes()) {
-            let _ = headers.insert(INNER_ERROR_CODE, code);
+            let _ = headers.insert(GREPTIME_ERROR_CODE, code);
```

Comment:
Looks like the code is a string just them in https://greptimedb.rs/common_error/status_code/enum.StatusCode.html

Prefer to return the integer code instead.

https://greptimedb.rs/src/common_error/status_code.rs.html#21-88

The integer code is unchanged for ever, but the name may be changed. And integer code is more friendly for error processing.